provider_sources/xyzservices-providers.json

```diff
@@ -373,5 +373,98 @@
             "max_zoom": 12,
             "name": "MapTiler.Terrain"
         }
+    },
+    "OrdnanceSurvey": {
+        "Road_3857": {
+            "url": "https://api.os.uk/maps/raster/v1/zxy/Road_3857/{z}/{x}/{y}.png?key={key}",
+            "html_attribution": "Contains OS data &copy Crown copyright and database right {year}",
+            "attribution": "Contains OS data (C) Crown copyright and database right {year}",
+            "key": "<A valid OS MapsAPI Key. Get a free key here - https://osdatahub.os.uk/>",
+            "crs": "EPSG:3857",
+            "min_zoom": 7,
+            "max_zoom": 16,
+            "max_zoom_premium": 20,
+            "bounds": [[-841264.4659119569, 6405988.404624734], [404617.877362011, 8733428.658192052]],
```

Comment:
I suppose that bounds for the default 3857 CRS should actually be in lat lon so you can directly pass them to leaflet and similar (https://leafletjs.com/reference.html#latlngbounds) as documented in our [contributing guide](https://xyzservices.readthedocs.io/en/stable/contributing.html).

Reply:
The bounds shall be in latitude longitude coordinates (in degrees), not in the unit of 3857 projection. That is what I was trying to say. The 3857 is the default for the XYZ standard and applications assume 3857 of tiles but bounds in lat long.

Reply:
I made a mistake in projecting from BNG.
BNG = 0,0,7e5,13e5
3857 = -9.496386752661158, 49.76680723514262, 3.634744234622926, 61.465188864662096

Should I be including these bounds?
They're not explicitly stated for 3857, only BNG.

Reply:
Yes, include `49.76680723514262, -9.496386752661158, 61.465188864662096, 3.634744234622926` for those tiles that are in 3857. Maybe with a bit less decimals :). 

Bounds for those in 27700 can stay as they are like we have in cases of other custom projections.

Reply:
Rounded to 9sf like SwissFederalGeoportal:  `49.766807, -9.496386, 61.465189, 3.634745`